app/assets/stylesheets/components/_border.scss

```diff
@@ -10,6 +10,9 @@
 .rounded-xl { border-radius: $border-radius-xl; }
 .rounded-xxl { border-radius: $border-radius-xxl; }
 
+.border-blue-lighter { border-color: $blue-lighter; }
```

Comment:
This doesn't look like it's being used, and after talking it through with @rtwell this can also just fall back to using the default `$border-color`.